src/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+import './styles/index.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please do not disable eslint checks. Eslint helps improve your Javascript's quality and keeps it to standard.